### DISCUSSION TITLE
Will Teslausb work with the new Model 3 Highland?

### DISCUSSION THREAD

**Discussion by jakubgt** [marcone/teslausb#847] [Q&A]
Will Teslausb work with the new Model 3 Highland? I have a new model 3 performance coming in, I was wondering if anyone had any luck having this work on their highland model

**Comment by m24andrew** [comment]
I have a 2024 Model 3 SR+ and all (recent clips from one journey) working well although I only started this morning.  No saved or sentry footage but so far so good. Uploading to Dropbox too and using the Pi Zero W 2...only issue was making sure I had the right cable as with the first one, the car didn't recognise the Pi.
